source/fe/fe_bernardi_raugel.cc

```diff
@@ -43,7 +43,7 @@ FE_BernardiRaugel<dim>::FE_BernardiRaugel(const unsigned int p)
       FiniteElementData<dim>(get_dpo_vector(),
                              dim,
                              2,
-                             FiniteElementData<dim>::Hdiv),
+                             FiniteElementData<dim>::H1),
```

Comment:
Is this correct? Is the element in fact continuous?

Reply:
Depending on how the DOFs on the edges are handled, yes it could be continuously differentiable on the entire mesh.